### PR TITLE
bounty: added progress bar

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description
- Added progress bar for expiring time
- Refactored a bit

x

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- css
- js

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@PixelantDesign As promised ^_^ (Used green instead of blue, we can change it back to blue -> if you want that )

**Comment by thelostone-mc** [comment]
Mobile needed some breathing space! Missed that  :P 

screen shot 2018-04-02 at 4 23 38 pm

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -164,13 +164,26 @@ var callbacks = {
 
     expires_date = new Date(val);
     now = new Date(result['now']);
-    var response = timeDifference(now, expires_date);
 
-    if (new Date(val) < new Date()) {
+    var percent = 100 * (
```
Comment (mkosowsk) [inline app/assets/v2/js/pages/bounty_details.js]: Please forgive the nitpick, but would something like percentCompleted be a bit more explicit? 😛

**Comment by mkosowsk** [comment]
Raised a super minor naming nitpick but this LGTM 👍🏻 Didn't test tho

File: app/assets/v2/js/pages/bounty_details.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: @mkosowsk kept it as `expiringInPercentage`. 
fixing this before @mbeacom sees this and judges me :P

Comment (mkosowsk) [reply app/assets/v2/js/pages/bounty_details.js]: :)

Comment (mbeacom) [reply app/assets/v2/js/pages/bounty_details.js]: 🤷‍♂️  No judgement here!

File: app/assets/v2/js/pages/bounty_details.js [hunk 2]
```diff
@@ -165,11 +165,11 @@ var callbacks = {
     expires_date = new Date(val);
     now = new Date(result['now']);
 
-    var percent = 100 * (
+    var expiringInPercentage = 100 * (
```
Comment (mbeacom) [inline app/assets/v2/js/pages/bounty_details.js]: 👨‍⚖️

Comment (mkosowsk) [reply app/assets/v2/js/pages/bounty_details.js]: 😍

**Comment by thelostone-mc** [comment]
@PixelantDesign Nooo, not in progress :P This is all done

**Comment by mbeacom** [comment]
@thelostone-mc Would you mind resolving the `stylelint` errors?

**Comment by thelostone-mc** [comment]
@mbeacom Done!

**Comment by thelostone-mc** [comment]
> Pixelant [2:18 PM]
> hi thanks

> https://github.com/gitcoinco/web/pull/764
for that one i told him we dont need a bar for expired time
we should remove the bar and just show when it was finished

Taken care of!

x


@mbeacom @PixelantDesign

**Pull request merged**